### PR TITLE
Buff lights

### LINKED ISSUE #448: Buff lights
With the new lightning system, a lot of light sources could need a boost in their light power. Things like floodlights outputs almost no light.

### PR DESCRIPTION
**Pull request by halworsen**
To resolve #448!

### CODE REVIEW

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -309,12 +310,12 @@ var/global/list/obj/item/device/pda/PDAs = list()
 /obj/item/device/pda/pickup(mob/user)
 	if(fon)
 		set_light(0)
-		user.set_light(user.light_range + f_lum)
+		user.set_light(user.light_range + f_lum, user.light_power + f_pow)
```
Comment (Kwask) [inline code/game/objects/items/devices/PDA/PDA.dm]: Check out flashlight code for this. Setting the user light level in this way is prone to failure.

File: code/game/machinery/floodlight.dm [hunk 2]
```diff
@@ -59,7 +61,7 @@
 			return
 		on = 1
 		user << "\blue You turn on the light"
-		set_light(brightness_on)
+		set_light(brightness_range, brightness_power, light_color)
```
Comment (Kwask) [inline code/game/machinery/floodlight.dm:63]: The issue with floodlights is their light color, not the range of the light. It was set to look whiter than normal light originally, but when we added the new light system, it became very dim. Removing the light color should fix this problem.

Comment (halworsen) [inline code/game/machinery/floodlight.dm:63]: Even without the light color, [they're a bit dim](http://vgy.me/erVvJP.png).

Comment (halworsen) [inline code/game/machinery/floodlight.dm:63]: I settled on buffing the light power one notch, while keeping the range unchanged. Light color's been removed.

**Comment by lmas** [comment]
Only need to buff the mining lanters too? Are you ready for a merge then?

**Comment by halworsen** [comment]
Yep, only lanterns left. I've been away from a computer that's capable of compiling in a reasonable amount of time, so I haven't gotten to it yet.

I'll finish and merge this today.

**Comment by lmas** [comment]
:+1: No sweat, just checking in

**Pull request merged**